src/components/services/Events.vue

```diff
@@ -1,39 +1,113 @@
+<script>
+  import CustomHeader from '@/components/shared/CustomHeader.vue';
+  export default {
+    components: {
+      CustomHeader,
+    },
+    data() {
+       return {
+          items: [
```

Comment:
It's good to be precise when naming variables. In this case, I would call this array 'events'. The only time you want to use something generic like 'items' is in a situation where you might have different kinds of objects passed in and you don't know ahead of time which objects they'll be.